test/integration/async.js

```diff
@@ -286,3 +301,5 @@ describe('Async: Cancel test result ', function () {
     assert.strictEqual(data.name, 'Google')
   })
 })
+
+it('Async: Import a test')
```

Comment:
should there be more here?

Reply:
Yeah there should, I left those as a reminder to set up integration tests, however I don't have the account details. Thanks for the ping...